### PR TITLE
Add Swarm Foundation grants to community grants page

### PR DESCRIPTION
**Pull request by costgallo**
Added Swarm Foundation Grants program

### CODE REVIEW

File: src/content/community/grants/index.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ These programs support the broad Ethereum ecosystem by offering grants to a wide
 
 These projects have created their own grants for projects aimed at developing and experimenting with their own technology.
 
+- [Swarm Foundation Grants Program]([url](https://my.ethswarm.org/grants)) - [Swarm Foundation]([url](https://www.ethswarm.org/)) _Grants Program_
```
Comment (minimalsm) [inline src/content/community/grants/index.md:29]:
```suggestion
- [Swarm Foundation Grants Program](https://my.ethswarm.org/grants) - [Swarm Foundation](https://www.ethswarm.org/) _Grants Program_
```

**Comment by minimalsm** [comment]
@all-contributors please add @costgallo for docs

**Comment by costgallo** [comment]
Hello, what is it missing to have it go live?

> @minimalsm
> 
> I've put up [a pull request](https://github.com/ethereum/ethereum-org-website/pull/11493) to add @costgallo! 🎉

Hello :) Is there anything from my side necessary to have it go live?

**Comment by costgallo** [comment]
> @minimalsm
> 
> I've put up [a pull request](https://github.com/ethereum/ethereum-org-website/pull/11493) to add @costgallo! 🎉

Hello :) is there anything necessary from my side to have it go live?

**Review by minimalsm — APPROVED** [review]

**Pull request merged**